kong/core/reports.lua

```diff
@@ -181,6 +185,15 @@ local function add_ping_value(k, v)
 end
 
 
+local function add_immutable_value(k, v)
+  v = get_report_value(v)
+  if v then
```

Comment:
This needs to be `if v ~= nil`, otherwise we don't respect the above behavior of reports, and we could missing some values when they are `false`.